common/djangoapps/entitlements/api/v1/views.py

```diff
@@ -231,8 +227,6 @@ class EntitlementEnrollmentViewSet(viewsets.GenericViewSet):
         - Unenroll
         - Switch Enrollment
     """
-    authentication_classes = (JwtAuthentication, SessionAuthentication,)
```

Comment:
Would there ever be a case where someone actually wanted `SessionAuthentication` over `SessionAuthenticationAllowInactiveUser`?  What would they do, and how would they know they need to do it?

Same question for `OAuth2AuthenticationAllowInactiveUser` which everyone gets as well (I guess).